src/utils/api.js

```diff
@@ -7,112 +7,24 @@
-        "url": "https://www.dslreports.com/forum/r31901625-New-BGW-210-700-Firmware-1-5-11"
-    },
-    "16982986": {
-        "by": "qiuyesuifeng",
-        "descendants": 63,
-        "id": 16982986,
-        "score": 160,
-        "time": 1525315440,
-        "title": "GitHub's online schema migration for MySQL",
-        "type": "story",
-        "url": "https://github.com/github/gh-ost"
-    },
-    "16982396": {
-        "by": "duncanmeech",
-        "descendants": 43,
-        "id": 16982396,
-        "score": 216,
-        "time": 1525307934,
-        "title": "The life cycle of HIV in 3D [video]",
-        "type": "story",
-        "url": "https://blogs.scientificamerican.com/observations/watch-the-life-cycle-of-hiv-in-colorful-new-detail/"
-    },
-    "16978932": {
-        "by": "vanni",
-        "descendants": 327,
-        "id": 16978932,
-        "score": 585,
-        "time": 1525280441,
-        "title": "Python startup time: milliseconds matter",
-        "type": "story",
-        "url": "https://mail.python.org/pipermail/python-dev/2018-May/153296.html"
-    },
-    "16978901": {
-        "by": "zengid",
-        "descendants": 248,
-        "id": 16978901,
-        "score": 517,
-        "time": 1525280236,
-        "title": "Proton Native ? React Native for the desktop",
-        "type": "story",
-        "url": "https://proton-native.js.org/"
-    },
-    "16981439": {
-        "by": "hapnin",
-        "descendants": 133,
-        "id": 16981439,
-        "score": 214,
-        "time": 1525297588,
-        "title": "Goldman Sachs to Open a Bitcoin Trading Operation",
-        "type": "story",
-        "url": "https://nytimes.com/2018/05/02/technology/bitcoin-goldman-sachs.html"
-    },
-    "16975853": {
-        "by": "dankohn1",
-        "descendants": 0,
-        "id": 16975853,
-        "score": 62,
-        "time": 1525260181,
-        "title": "Humpback Whale Baby Boom Near Antarctica",
-        "type": "story",
-        "url": "https://www.nytimes.com/2018/05/01/science/humpback-whale-antarctica.html"
-    },
-    "16978544": {
-        "by": "standeven",
-        "descendants": 198,
-        "id": 16978544,
-        "score": 364,
-        "time": 1525278211,
-        "title": "'Anti-authority' tech rebels take on ISPs, connect NYC with cheap Wi-Fi",
-        "type": "story",
-        "url": "http://www.cbc.ca/news/technology/wifi-nyc-mesh-new-york-city-1.4617106"
-    }
-};
+const BASE_URL="https://hacker-news.firebaseio.com/v0";
+const fetchJson = url => fetch(url).then(res => res.json());
+const first10 = arr => arr.slice(0, 10);
+
+
+const getItemsIds = () => {
+    return fetchJson(`${BASE_URL}/topstories.json`)
+        .then(data => first10(data));
+}
+
+const getItem = (id) => {
+   return fetchJson(`${BASE_URL}/item/${id}.json`);
+}
+
 
 export const api = {
-    getItemsIds: () => itemsIds,
-    getItem: (id) => itemsPerId[id],
+    getItemsIds,
+    getItem,
+   // getItem: (id) => itemsPerId[id],
```

Comment:
comments are dead code, strive to not having them. you have git to store prev versions of your code

Reply:
removed